dandiapi/zarr/tests/test_zarr.py

```diff
@@ -334,7 +334,7 @@ def test_zarr_rest_delete_multiple_files(
 
     # Assert not found
     for file in zarr_files:
-        assert not zarr_archive.storage.exists(zarr_archive.s3_path(file))
```

Comment:
Based on the object types, it seems the old use of `file` instead of `file.path` here was a bug.